rollbar/contrib/django/middleware.py

```diff
@@ -18,11 +18,16 @@
 
 import rollbar
 
+import django
 from django.core.exceptions import MiddlewareNotUsed
-from django.core.urlresolvers import resolve
 from django.conf import settings
 from django.http import Http404
 
+if django.VERSION >= (1, 10):
```

Comment:
Hmm, I'm not sure if this is the right way to do comparisons or not. `django.VERSION` usually has text in it. For example, `(1, 11, 1, u'final', 0)`.

I think it might be easier to do an import check similar to how we check for `MiddlewareMixin` below.